.codeclimate.yml

```diff
@@ -20,6 +20,9 @@ plugins:
         enabled: false
       DeclarationOrder:
         enabled: false
+      NestingDepth:
+        max_depth: 5
```

Comment:
Now there is a `max_depth` of 3. I think there are reasons for allowing a bit deeper level of nesting. Cascade is part of CSS and there are occasions where we need to influence global styles. like for example this:

``` scss
.admin {
  &__section-header {
    .ofn-drop-down {
      &:hover,
      &.expanded {
         // I need to override this global styles
      }
  }
}
```

Another way of doing this is by using `!important`. But I guess nobody wants to go this way right? ;) Specificity is better than using `!important`